integrations/symfony-bundle.rst

```diff
@@ -56,6 +56,44 @@ Web Debug Toolbar
 
 When using a client configured with ``HttplugBundle``, you will get debug information in the web debug toolbar. It will tell you how many request were made and how many of those that were successful or not. It will also show you detailed information about each request.
 
+You can configure the bundle to show debug information for clients found with discovery. You may also force a specific client to be found when a third party library is using discovery. The configuration below makes sure the client with service id ``httplug.clients.my_guzzle5`` is returned when calling ``HttpClientDiscovery::find()`` . It does also make sure to show debug info for asynchronous clients.
+
+.. code-block:: yaml
+
+    httplug:
+        clients:
+            my_guzzle5:
+                factory: 'httplug.factory.guzzle5'
+        discovery:
+            client: 'httplug.clients.my_guzzle5'
+            async_client: 'auto'
+
+The debug info for normal HTTP clients are enabled by default but not for async clients. You can turn all debug info off for auto discovered clients by setting the value to ``false``.
```

Comment:
lets try to make this more readable:

For normal clients, the auto discovery debug info is enabled by default. For async clients, debug is not enabled by default to avoid errors when using the bundle with a client that can not do async. To get debug information for async clients, set ``discovery.async_client`` to ``'auto'`` or an explicit client.

You can turn off all interaction of the bundle with auto discovery by setting the value of ``discovery.client`` to ``false``.